opts/ip_test.go

```diff
@@ -0,0 +1,54 @@
+package opts
+
+import (
+	"net"
+	"testing"
+)
+
+func TestIpOptString(t *testing.T) {
```

Comment:
integration and opensource project is about collaborating with other Dev teams or the likes. There is no point of stealing a code that is being work on... Progression is real and not a vitual visibility that just happens by chance and we are not reboots....